java/samples/sample-code/src/main/java/com/microsoft/semantickernel/samples/syntaxexamples/Example09_FunctionTypes.java

```diff
@@ -377,18 +461,18 @@ public Mono<Void> InputStringTaskWithVoidResult(
             name = "noInputComplexReturnTypeAsync",
             returnType = "java.time.temporal.Temporal"
         )
-        public Mono<LocalDateTime> noInputComplexReturnTypeAsync() {
+        public Mono<OffsetDateTime> noInputComplexReturnTypeAsync() {
             return Mono.just(
-                LocalDateTime.now()
+                OffsetDateTime.now()
             );
         }
 
         @DefineKernelFunction(
             name = "noInputComplexReturnType",
             returnType = "java.time.temporal.Temporal"
         )
-        public LocalDateTime noInputComplexReturnType() {
-            return LocalDateTime.now();
+        public OffsetDateTime noInputComplexReturnType() {
```

Comment:
Why did you choose OffsetDateTime here as opposed to ZonedDateTime?

Reply:
This is purely as our DateTimeContextVariable works on OffsetDateTime, if we add a converter for Local or Zoned date time we could use that here